### ISSUE TITLE
pre-training weight

### ISSUE THREAD

**Issue by nh945** [Accioy/SAR2optical#2]
Thank you for your sharing. Could you provide the pre-training weight? It cannot run perfectly because of computer reasons,Looking forward to your reply！

**Comment by Accioy** [comment]
Sorry but I've been out of this field for a long time and I didn't preserve the weights.

**Comment by nh945** [comment]
Thank you for your reply! If I use 20000 pairs of pictures for training, can I achieve the effect? My ACC has reached 100%, but GEN_loss has been fluctuating, is it normal? Sincerely wish you do better in other  field

**Comment by nh945** [comment]
> Sorry but I've been out of this field for a long time and I didn't preserve the weights.

Thank you for your reply! I am training this code. I have used 400,000 pictures but trained 24 epochs, but the results have been fluctuating. The network conversion result is not clear, because the training cost is high, I am considering whether to continue training. Can you give me some advice? Like what code to change? Or wait for a long training session? Sincerely wish you do better in other field！Looking forward to your reply.

**Comment by Accioy** [comment]
I forget how many pictures I used but I got the best result with 150 epochs. The loss should converge stably. Maybe you can try to use less pictures and train more epochs, then check the predictions in both trainset and testset.
